app/services/trading_strategies.py

```
@@ -4785,50 +4785,18 @@ def safe_int(value: Any, default: int = 0) -> int:
-                "volatility_is_percent": False,
-                "max_drawdown_is_percent": True,
-                "win_rate_is_percent": True,
-                "average_trade_is_percent": True,
-                "largest_win_is_percent": True,
-                "largest_loss_is_percent": True
+                "data_quality": "no_data",
+                "status": "no_data_available",
+                "message": "No verified trades or backtests found for requested period",
+                "total_trades": 0,
             }
         except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)